features/step_definitions/custom_web_steps.rb

```diff
@@ -288,6 +288,14 @@ def take_screenshots_with_login
   find("#bitcoin_address")['value'].should == "AAAAAA"
 end
 
+Given /^I have configured a Liberapay username$/ do
+  AppConfig.settings.liberapay.username = "BBBBBB"
+end
+
+Then /^I should see the Liberapay donate button$/ do
+  find("#liberapay-button")['href'].should == "https://liberapay.com/BBBBBB/donate"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.